XIVComboExpanded/CustomComboPreset.cs

```diff
@@ -1343,9 +1351,17 @@ public enum CustomComboPreset
     [IconsCombo([NIN.TenChiJin, UTL.ArrowLeft, NIN.Meisui, UTL.Blank, NIN.Buffs.ShadowWalker, UTL.Checkmark])]
     [SectionCombo("Ninjutsu features")]
     [ExpandedCustomCombo]
+    [ConflictingCombos(NinjaDokumoriConsolidationCombo)]
     [CustomComboInfo("Ten Chi Jin to Meisui", "Replace Ten Chi Jin (the move) with Meisui while Suiton is up.\nCooldown tracking plugin recommended.", NIN.JobID)]
     NinjaTCJMeisuiFeature = 3005,
 
+    [IconsCombo([NIN.Dokumori, UTL.ArrowLeft, NIN.TenChiJin, UTL.ArrowLeft, NIN.Ten, UTL.ArrowLeft, NIN.Chi, UTL.ArrowLeft, NIN.Jin, UTL.ArrowLeft, NIN.Meisui])]
+    [SectionCombo("Ninjutsu features")]
+    [ExpandedCustomCombo]
+    [ConflictingCombos(NinjaTCJMeisuiFeature)]
+    [CustomComboInfo("2 Minute Cooldowns Consolidation", "Consolidate Dokumori, TenChiJin (The Move), its combo steps, and Meisui with eachother.", NIN.JobID)]
+    NinjaDokumoriConsolidationCombo = 3025,
```

Comment:
Similarly, I think this could be a sub-combo of the existing `NinjaTCJMeisuiFeature` combo, since it _also_ includes Meisui.  However, it should aksi definitely be an Accessibility rather than Expanded type combo, given how much it is consolidating.
```suggestion
    [IconsCombo([NIN.Dokumori, UTL.ArrowLeft, NIN.TenChiJin, UTL.ArrowLeft, NIN.Ten, UTL.ArrowLeft, NIN.Chi, UTL.ArrowLeft, NIN.Jin, UTL.ArrowLeft, NIN.Meisui])]
    [SectionCombo("Ninjutsu features")]
    [AccessibilityCustomCombo]
    [ParentCombo(NinjaTCJMeisuiFeature)]
    [CustomComboInfo("2 Minute Cooldowns Consolidation", "After Dokumori is used, replace it with Ten Chi Jin (the action), its mudra steps, Meisui, and Tenri Jindo in sequence.", NIN.JobID)]
    NinjaDokumoriConsolidationCombo = 3025,
```